code/jumpnode/jumpnode.cpp

```diff
@@ -56,14 +57,18 @@ CJumpNode::CJumpNode(vec3d *position) : m_radius(0.0f), m_modelnum(-1), m_objnum
     flagset<Object::Object_Flags> default_flags;
     default_flags.set(Object::Object_Flags::Renders);
     m_objnum = obj_create(OBJ_JUMP_NODE, -1, -1, NULL, &m_pos, m_radius, default_flags);
+
+	// And finally set up animation info for possible instrinsic_rotate
+	m_polymodel_instance_num = model_create_instance(false, m_modelnum);
```

Comment:
The model has been loaded at this point, so you can check the model flags to see whether `PM_FLAG_HAS_INTRINSIC_ROTATE` is set before creating the instance.